Gridy/View/WhiteLayerView.swift

```diff
@@ -26,14 +26,15 @@ class WhiteLayerView: UIView {
         let width = self.bounds.width
         let height = self.bounds.height
         var drawRectangle = CGRect()
+        let gridFrameView = GridFrameView()
```

Comment:
You don't want to create a new instance but use the one displayed on screen.

Reply:
Minor feedback: `gridFrameView` doesn't have to be a `GridFrameView` type since this is a simple placeholder view with constraint that serve as a drawing guide.